ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -81,7 +81,12 @@ class VariableProcessorTest {
 
     @Test
     fun `process variables processes price_per_period`() {
-        expectVariablesResult("{{ price_per_period }}", "PRICE_PER_PERIOD")
+        expectVariablesResult("{{ price_per_period }}", "$67.99/yr")
```

Comment:
We should add tests for other period units. Will add those for this and other tests in a followup PR.